lib/search.rb

```diff
@@ -496,6 +482,30 @@ def self.advanced_filters
 
   private
 
+    def search_tags(posts, match)
+      return if match.nil?
+
+      if match.include?('+')
+        tags = match.split('+')
+        posts.where("topics.id IN (
+              SELECT tt.topic_id
```

Comment:
Minor nitpick but lets indent the code like so:
```
posts.where("topics.id IN (
  SELECT DISTINC...
  FROM
  WHERE
")
```

Reply:
OK.